admin/load.php

```diff
@@ -339,6 +341,87 @@ function perflab_get_module_data( $module_file ) {
 	return $module_data;
 }
 
+/**
+ * Initialise Admin Pointer
+ *
+ * Handles the bootstrapping of the admin pointer.
+ * Mainly jQuery code that is self-initialising.
+ *
+ * @param string $hook_suffix The current admin page.
+ * @since 1.0.0
+ */
+function perflab_admin_pointer( $hook_suffix ) {
+
+	if ( ! in_array( $hook_suffix, array( 'index.php', 'plugins.php' ), true ) ) {
+		return;
+	}
+
+	$current_user = get_current_user_id();
+	$dismissed    = explode( ',', (string) get_user_meta( $current_user, 'dismissed_wp_pointers', true ) );
+
+	if ( in_array( PERLAB_ADMIN_POINTER, $dismissed, true ) ) {
```

Comment:
See above, I think it's okay to duplicate the name here. While that's code duplication, it's only used twice in the entire code, and introducing a global constant for it is worse IMO.
```suggestion
	if ( in_array( 'perflab-admin-pointer', $dismissed, true ) ) {
```

If we wanted to also avoid the duplication, an alternative would be to pass the pointer in a variable to the `perflab_render_pointer( $pointer_id )` method. But that would also come with drawbacks since then we'd need to make the filter callback an anonymous function, which isn't really "WP core-friendly".